src/components/sellLuggageAllowance/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import {withRouter} from 'react-router-dom';
+import './style.scss'
+//components
+import SellInfo from '../sellInfo'
+
+class SellLuggageAllowance extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchParams: {
+                to: '',
```

Comment:
Instead of having "to" and "from", we can just let the user input flight number and date as we have a flight database to pull flight details from